.woodpecker.star

```diff
@@ -2260,7 +2260,6 @@ def skipIfUnchanged(ctx, type):
         "README.md",
     ]
     unit = [
-        "**/*_test.go",
```

Comment:
What is the reasoning behind that change?

Reply:
It's just a test. We're experiencing issues with the `exlcude` feature since your latest change.